build/local/server/docker-compose.test.yaml

```diff
@@ -18,6 +18,8 @@ services:
       - database
     networks: 
       - sherlock-local-dev
+    volumes:
+      - $PWD:/build
```

Comment:
I know we need the build folder, not actually sure why we need it in volumes, but I'm just assuming we do now.

Reply:
it's so the coverage report is accessible to the codecov action step in the pipeline. Otherwise we'd loose it when the container stops.